templates/kubernetes/managed.html

```diff
@@ -47,7 +47,7 @@ <h2 class="p-heading--three">24 hours a day, everyday</h2>
 
   <section class="p-strip is-bordered">
     <div class="row">
-      <h2>Getting started with Canonical Distribution of Kubernetes</h2>
+      <h2>Getting started with Canonical Distribution of Kubernetes <sup class="p-heading--five">® <a href="#fn1" id="r1">*</a></sup></h2>
```

Comment:
I think the symbol should be escaped and add an encoded space so that they stick to the word Kubernetes
`&reg;&nbsp`

Reply:
Done